mp4parse_capi/src/lib.rs

```diff
@@ -743,8 +748,13 @@ fn create_sample_table(track: &Track) -> Option<Vec<mp4parse_indice>> {
     }
 
     // Mark the sync sample in sample_table according to 'stss'.
-    for iter in &stss.samples {
-        sample_table[(iter - 1) as usize].sync = true;
```

Comment:
Obviously, my previous test stream is not good enough, so these problems doesn't be found.